state_management/javascript/sdk/order-processor/index.js

```diff
@@ -1,45 +1,41 @@
-import { DaprClient } from 'dapr-client'; 
+import { DaprClient } from 'dapr-client'
 
-const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
-const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
+const DAPR_HOST = process.env.DAPR_HOST ?? "http://localhost"
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT ?? "3500"
+const DAPR_STATE_STORE_NAME = "statestore"
 
 async function main() {
-    for(var i = 1; i <= 10; i++) {
-        const orderId = i;
-        const order = {orderId: orderId};
-        const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT);
-        const STATE_STORE_NAME = "statestore";
-
-        // Save state into the state store
-        client.state.save(STATE_STORE_NAME, [
+    const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT)
+
+    // For each loop, Save order, Get order, and Delete order
+    for (var i = 1; i <= 10; i++) {
+        const orderId = i.toString()
+        const order = { orderId }
+        const state = [
             {
-                key: orderId.toString(),
+                key: order.orderId.toString(),
```

Comment:
use orderId instead of order.orderId?

Reply:
Won't fix. It's less robust. See other comment for more details.